tests/integration/devfile/utils/utils.go

```diff
@@ -186,3 +189,81 @@ func ExecWithRestartAttribute(projectDirPath, cmpName, namespace string) {
 	Expect(output).To(ContainSubstring("if not running"))
 
 }
+
+// OdoWatch creates files, dir in the context and watches for the changes to be pushed
+// this was taken from e2e_images_test.go and modified for devfile
+func OdoWatch(cmpName, project, context string, stringsToBeMatched []string, runner interface{}, platform string) {
+
+	startSimulationCh := make(chan bool)
+	go func() {
+		startMsg := <-startSimulationCh
+		if startMsg {
+			err := os.MkdirAll(filepath.Join(context, ".abc"), 0750)
+			if err != nil {
+				panic(err)
```

Comment:
I think we shouldn't panic as it will cause the test suite to panic and no subsequent tests will run. You can use gingkoRecover() to recover.

Reply:
I'll just use `Expect(err).To(BeNil())` like the other test cases in odo